client/src/commands/testSmartContractCommand.ts

```diff
@@ -223,13 +223,42 @@ export async function testSmartContract(chaincode?: InstantiatedContractTreeItem
         return;
     }
 
-    // If TypeScript, update JavaScript Test Runner user settings
+    // If TypeScript, update JavaScript Test Runner user settings and create tsconfig.json
     if (testLanguage === 'TypeScript') {
         const runnerArgs: string = vscode.workspace.getConfiguration().get('javascript-test-runner.additionalArgs') as string;
         if (!runnerArgs || !runnerArgs.includes('-r ts-node/register')) {
             // If the user has removed JavaScript Test Runner since generating tests, this update will silently fail
             await vscode.workspace.getConfiguration().update('javascript-test-runner.additionalArgs', '-r ts-node/register', vscode.ConfigurationTarget.Global);
         }
+
+        // Setup Tsconfig path
+        const tsConfigPath: string = path.join(localSmartContractDirectory, 'tsconfig.json');
+
+        // Tsconfig file content
+        const tsConfigContents: any = {
+            compilerOptions: {
+                outDir: 'dist',
+                target: 'es2017',
+                moduleResolution: 'node',
+                module: 'commonjs',
+                declaration: true,
+                sourceMap: true
+            },
+            include: [
+                './functionalTests/**/*'
+            ],
+            exclude: [
+                'node_modules'
+            ]
+        };
+
+        // Try to create tsconfig.json file
+        try {
+            await fs.writeJson(tsConfigPath, tsConfigContents, {spaces: '\t'});
```

Comment:
what happens if one already exists, we should probably only create one if it doesn't exist

Reply:
We now check if a `tsconfig.json` exists. If it already exists, we warn the user but continue and assume that the config will work.